bftengine/src/preprocessor/tests/preprocessor_test.cpp

```diff
@@ -105,7 +106,7 @@ DummyRequestsHandler requestsHandler;
 
 class DummyReplica : public InternalReplicaApi {
  public:
-  explicit DummyReplica(bftEngine::impl::ReplicasInfo repInfo) : replicasInfo_(move(repInfo)) {}
+  explicit DummyReplica(bftEngine::impl::ReplicasInfo& repInfo) : replicasInfo_(move(repInfo)) {}
```

Comment:
why was it important to not pass by value here?
The new sematic seems weird, you have a lvalue ref (I think you meant to use a rvalue ref) from which you take ownership. A caller wouldn't expect his object to be nullified after passing it to DummyReplica. This was not the case with the parameter being passed by value.

Reply:
tidy check complained, I'll fix it

Reply:
fixed